src/Braintree/Braintree.csproj

```diff
@@ -17,8 +17,7 @@
       * Add `Username` and `VenmoUserId` to `PaymentMethodNonceDetails`.
     </PackageReleaseNotes>
     <PackageProjectUrl>https://github.com/braintree/braintree_dotnet</PackageProjectUrl>
-    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.3' ">$(PackageTargetFallback);dnxcore50;portable-net45+win7</PackageTargetFallback>
```

Comment:
@iamcarbon @gitvinit sorry I'm dragging my feet on this one, I just want to make sure we're not losing core 1.X or `net451` support here. It seems like this line _needs_ to be removed for netstandard packages, but that kicks the minimum framework version up to `net461` (I think). Thanks for your patience.